tests/import_export/test_export_to_geff.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+import zarr
+
+from funtracks.data_model.tracks import Tracks
+from funtracks.import_export.export_to_geff import export_to_geff
+
+
+@pytest.mark.parametrize("ndim", [2, 3])
+@pytest.mark.parametrize("track_type", (Tracks,))
```

Comment:
Did you try with SolutionTracks too and it didn't work? Or you decided it was unnecessary?